UnitTests/BraintreeThreeDSecureTests/BTThreeDSecureClient_Tests.swift

```diff
@@ -276,7 +276,7 @@ class BTThreeDSecureClient_Tests: XCTestCase {
             expectation.fulfill()
         }
 
-        waitForExpectations(timeout: 4)
+        waitForExpectations(timeout: 10)
```

Comment:
Similar to the UI tests, many of these tests are waiting on a result from the cardinal `setup` call. In the future, we should consider mocking the Cardinal API call (which may prove to be difficult). A number of our 3DS tests rely on making actual network requests either in our SDK or the Cardinal SDK. We should consider revisiting these test as a whole to determine what can be mocked in this class.

Reply:
➕ 1 - our unit tests should not make any network requests / should work fine without internet

Reply:
I created a new branch off of this PR to mock the class where the setup call is made - if we like this approach to mock the setup step, I can merge it into this branch: [mock-cardinal-setup-call](https://github.com/braintree/braintree_ios/compare/fix-flakey-tests...mock-cardinal-setup-call) - I was able to bump almost all of the timeouts back down.

Reply:
Merged these changes into the branch